### PR TITLE
rethinkdb 2.4.0

### PR DESCRIPTION
**Pull request by gabor-boros**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
-----

### CODE REVIEW

File: Formula/rethinkdb.rb [hunk 1]
```diff
@@ -15,33 +15,12 @@ class Rethinkdb < Formula
...
-    sha256 "a26e452ce1f16541a9ba40057af154e594fe89665c2c656994ceab103d2017e9"
-  end
-
   def install
     args = ["--prefix=#{prefix}"]
 
     # rethinkdb requires that protobuf be linked against libc++
     # but brew's protobuf is sometimes linked against libstdc++
-    args += ["--fetch", "protobuf"]
+    args += ["--fetch", "protobuf", "--allow-fetch"]
```
Comment (Moisan) [inline Formula/rethinkdb.rb]: From the configure doc:
> --allow-fetch           Allow fetching missing dependencies
This is not acceptable for Homebrew. We require to handle all dependencies outside of the build step to ensure reproducibility.

**Review by Moisan — CHANGES_REQUESTED** [review]

File: Formula/rethinkdb.rb (hunk 1 above)
Comment (gabor-boros) [reply Formula/rethinkdb.rb]: Hello Moisan,
Thank you for your reply. I’ll handle that soon.

**Comment by gabor-boros** [comment]
@Moisan I just adjusted the code based on what you wrote. Also, I was able to run the tests and they pass

**Comment by Moisan** [comment]
Thank you @gabor-boros for your first contribution! Without people like you, we couldn't run this project.

**Review by Moisan — APPROVED** [review]

**Comment by Moisan** [comment]
Closed with commit 9e9243a869c65cb586afd691f9743e52791bb8c6. (I removed by mistake an automatically generated line during the commit squashing)

**Comment by gabor-boros** [comment]
Thank you @Moisan for your support! It was a good experience to contribute to homebrew ☺️

**Pull request closed without merging**